lib/jsdom/living/nodes/Element-impl.js

```diff
@@ -278,6 +278,32 @@ class ElementImpl extends NodeImpl {
     attributes.removeAttributeByNameNS(this, namespace, localName);
   }
 
+  toggleAttribute(name, force) {
```

Comment:
Nit: s/name/qualifiedName/g?

Reply:
Yeah, I went with `name` to be consistent with the other methods in this file but I suppose we can change those as well later on.